api/android/build-android-lib.sh

```diff
@@ -148,15 +157,26 @@ check_package sed
 check_package zip
 
 # Android SDK (Set your own path)
-[ -z "$ANDROID_HOME" ] && echo "Need to set ANDROID_HOME." && exit 1
+if [[ -z $android_sdk_dir ]]; then
+    [ -z "$ANDROID_HOME" ] && echo "Need to set ANDROID_HOME." && exit 1
+    android_sdk_dir=$ANDROID_HOME
+fi
 
-echo "Android SDK: $ANDROID_HOME"
+if [[ -z $android_ndk_dir ]]; then
+    android_ndk_dir=$android_sdk_dir/ndk-bundle
```

Comment:
is it a concrete path `ndk-bundle`? In my case, ndk has not been installed there.

Reply:
https://developer.android.com/studio/command-line/variables

I will change to ANDROID_SDK_ROOT / ANDROID_NDK_ROOT